<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfacePer</class>
 <widget class="QMainWindow" name="interfacePer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1783</width>
    <height>1207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interfacePer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 81, 57)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>-1</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline;&quot;&gt;NextGenIT&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; text-decoration: underline;&quot;&gt;Gestion des magasins .&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom_magasin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-colorrgb:(186, 0, 0)</string>
    </property>
    <property name="text">
     <string>Id_magasin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type_de_magasin </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Emplacement</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Heures d'ouverture</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Date_de_creation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Téléphone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Heures de fermeture</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 0, 0)</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 0, 0)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 110, 52)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9_ajouter">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>430</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12_supprimer">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>741</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_id_supprimer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(68, 95, 71)</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_de_supp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Si vous voulez supprimer ou modifier , merci de donner L'ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>620</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recherche">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_critere">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>620</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rechercher :</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>emplacement</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_trie">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Trier :</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nom</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>type</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>emplacement</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_trier">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pdf">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_statistique">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>430</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistique</string>
    </property>
   </widget>
   <widget class="QWidget" name="chartViewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>370</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>296</width>
      <height>195</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_idCommande">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>250</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(68, 95, 71)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomClient">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>280</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(68, 95, 71)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_produit">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>310</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(68, 95, 71)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifierCommande">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>310</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>modifierC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimerCommande">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>280</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimerC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouterCommande">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>250</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ajouterC</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_etat">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>340</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Etat :</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EnAttente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Expediee</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Annulee</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit_dateCommande">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>370</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>250</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID-Cmd :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom-Cl :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom-Prd :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAfficherCARegion">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>C.A.R.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAfficherCAMagasin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>C.A.M.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_recommendProducts">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>AI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10_afficher">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCabine1">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>570</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cabine 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCabine2">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>600</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cabine 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAttente">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>630</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zone d'attente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_afficherCabines">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>640</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher l'historique des cabines</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1783</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
